pkg/middlewares/accesslog/field_middleware.go

```diff
@@ -72,3 +67,14 @@ func AddOriginFields(rw http.ResponseWriter, req *http.Request, next http.Handle
 	data.Core[OriginStatus] = capt.StatusCode()
 	data.Core[OriginContentSize] = capt.ResponseSize()
 }
+
+// InitServiceFields init service fields.
+func InitServiceFields(rw http.ResponseWriter, req *http.Request, next http.Handler, data *LogData) {
+	// Because they are expected to be initialized when the logger is processing the data table,
+	// the origin fields are initialized for when the response is returned by Traefik itself, and not a service.
```

Comment:
The "for" was intended ;-)

Reply:
So I don't understand the sentense

Reply:
I propose this substitution:

```suggestion
	// the origin fields are initialized in case the response is returned by Traefik itself, and not a service.
```